app/services/github_service.rb

```diff
@@ -41,6 +39,8 @@ def find_or_create_event_type
   end
 
   def handle_action
+    raise StandardError unless handleable_event?
```

Comment:
I thought that now we were "allowing" all the events even if some of them aren't handled now. What's the pointof raising an exception?

Reply:
for example this service is called by a job. If an exception is raised and the job is retried we'll be mistakenly creating multiple `Event`s